hr_attendance_modification_tracking/__manifest__.py

```diff
@@ -1,15 +1,17 @@
 # Copyright 2019 Creu Blanca
+# Copyright 2020 Landoo
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Hr Attendance Modification Tracking",
+    "name": "HR Attendance Modification Tracking",
     "summary": """
         Attendance changes will now be registered in the chatter.""",
-    "version": "13.0.1.0.0",
+    "version": "13.0.1.1.0",
```

Comment:
Next time don't bump version. I t is better to have the merge bot do that, it avoids code conflicts.